privaterelay/settings.py

```diff
@@ -379,6 +379,7 @@ def _get_initial_middleware() -> list[str]:
 SUBSCRIPTIONS_WITH_VPN = config("SUBSCRIPTIONS_WITH_VPN", default="", cast=Csv())
 
 MAX_ONBOARDING_AVAILABLE = config("MAX_ONBOARDING_AVAILABLE", 0, cast=int)
+MAX_ONBOARDING_FREE_AVAILABLE = config("MAX_ONBOARDING_FREE_AVAILABLE", 0, cast=int)
```

Comment:
suggestion (blocking): I think our `settings.py` default values should be the same that we use in `.env-dist`. I know other values don't match (e.g., `MAX_ONBOARDING_AVAILABLE` from which this was probably copied), but I think we should keep `.env-dist` and `settings.py` in sync going forward.

```suggestion
MAX_ONBOARDING_FREE_AVAILABLE = config("MAX_ONBOARDING_FREE_AVAILABLE", 3, cast=int)
```